pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -72,13 +79,6 @@
         create_ops_listC128,
     )
 
-    # pylint: disable=import-error, no-name-in-module, ungrouped-imports
-    from pennylane_lightning.core._serialize import (
-        QuantumScriptSerializer,
-        global_phase_diagonal,
-    )
-    from pennylane_lightning.core._version import __version__
-
```

Comment:
these headers changes again :sweat_smile:

Reply:
Hopefully fixed when #647 is merged.